modules/telco-core-crs-networking.adoc

```diff
@@ -9,27 +9,28 @@
-SR-IOV Network Operator,xref:../../telco_ref_design_specs/core/telco-core-ref-crs.adoc#telco-core-sriovsubscriptionopergroup-yaml[SriovSubscriptionOperGroup.yaml],No,No
+Component,Reference CR,Description,Optional
+Baseline,`Network.yaml`,"Configures the default cluster network, specifying OVN Kubernetes settings like routing via the host. It also allows the definition of additional networks, including custom CNI configurations, and enables the use of MultiNetworkPolicy CRs for network policies across multiple networks.",No
+Baseline,`networkAttachmentDefinition.yaml`,Optional. Defines a NetworkAttachmentDefinition resource specifying network configuration details such as node selector and CNI configuration.,No
+Load Balancer,`addr-pool.yaml`,Configures MetalLB to manage a pool of IP addresses with auto-assign enabled for dynamic allocation of IPs from the specified range.,No
+Load Balancer,`bfd-profile.yaml`,"Configures bidirectional forwarding detection (BFD) with customized intervals, detection multiplier, and modes for quicker network fault detection and load balancing failover.",No
+Load Balancer,`bgp-advr.yaml`,"Defines a BGP advertisement resource for MetalLB, specifying how an IP address pool is advertised to BGP peers. This enables fine-grained control over traffic routing and announcements.",No
+Load Balancer,`bgp-peer.yaml`,"Defines a BGP peer in MetalLB, representing a BGP neighbor for dynamic routing.",No
+Load Balancer,`community.yaml`,"Defines a MetalLB community, which groups one or more BGP communities under a named resource. Communities can be applied to BGP advertisements to control routing policies and change traffic routing.",No
+Load Balancer,`metallb.yaml`,Defines the MetalLB resource in the cluster.,No
+Load Balancer,`metallbNS.yaml`,Defines the metallb-system namespace in the cluster.,No
+Load Balancer,`metallbOperGroup.yaml`,Defines the Operator group for the MetalLB Operator.,No
+Load Balancer,`metallbSubscription.yaml`,Creates a subscription resource for the metallb Operator with manual approval for install plans.,No
+Multus - Tap CNI for rootless DPDK pods,`mc_rootless_pods_selinux.yaml`,Configures a MachineConfig resource which sets an SELinux boolean for the tap CNI plugin on worker nodes.,Yes
+NMState Operator,`NMState.yaml`,Defines an NMState resource that is used by the NMState Operator to manage node network configurations.,No
+NMState Operator,`NMStateNS.yaml`,Creates the NMState Operator namespace.,No
+NMState Operator,`NMStateOperGroup.yaml`,"Creates the Operator group in the openshift-nmstate namespace, allowing the NMState Operator to watch and manage resources.",No
+NMState Operator,`NMStateSubscription.yaml`,"Creates a subscription for the NMState Operator, managed through OLM.",No
+Node Tuning Operator, `tuned-queues.yaml`, "Configures the number of NIC queues for kernel level networking.", No
```

Comment:
where does this CR come from? 

/cc @imiller0

Reply:
This comes from here: https://github.com/openshift-kni/telco-reference/pull/108 - @MarSik this PR will be merged before GA, so I can keep this reference in the docs, correct?

Reply:
Yes, that is the plan. I am on it.